server_src/routes/system.js

```diff
@@ -16,43 +16,37 @@ const { fetchMongoDBConnectionString } = require("../app-env");
 const { isPm2, isNodemon, isNode } = require("../utils/env.utils.js");
 const { SettingsClean } = require("../lib/dataFunctions/settingsClean");
 
-router.get(
-  "/",
-  ensureAuthenticated,
-  ensureCurrentUserAndGroup,
-  async (req, res) => {
-    const clientSettings = await SettingsClean.returnClientSettings();
-    const serverSettings = await SettingsClean.returnSystemSettings();
-    const systemInformation = await SystemRunner.querySystemInfo();
-    const printers = Runner.returnFarmPrinters();
-    const softwareUpdateNotification =
-      softwareUpdateChecker.getUpdateNotificationIfAny();
-    let dashboardSettings =
-      clientSettings?.dashboard || getDefaultDashboardSettings();
+router.get("/", ensureAuthenticated, ensureCurrentUserAndGroup, async (req, res) => {
+  const clientSettings = await SettingsClean.returnClientSettings();
+  const serverSettings = await SettingsClean.returnSystemSettings();
+  const systemInformation = await SystemRunner.querySystemInfo();
+  const printers = Runner.returnFarmPrinters();
+  const softwareUpdateNotification = softwareUpdateChecker.getUpdateNotificationIfAny();
+  let dashboardSettings = clientSettings?.dashboard || getDefaultDashboardSettings();
 
-    res.render("system", {
-      name: req.user.name,
-      userGroup: req.user.group,
-      version: process.env[AppConstants.VERSION_KEY],
-      printerCount: printers.length,
-      page: "System",
-      octoFarmPageTitle: process.env[AppConstants.OCTOFARM_SITE_TITLE_KEY],
-      helpers: prettyHelpers,
-      clientSettings,
-      serverSettings,
-      systemInformation,
-      db: fetchMongoDBConnectionString(),
-      dashboardSettings: dashboardSettings,
-      serviceInformation: {
-        isDockerContainer: isDocker(),
-        isNodemon: isNodemon(),
-        isNode: isNode(),
-        isPm2: isPm2(),
-        update: softwareUpdateNotification
-      }
-    });
-  }
-);
+  res.render("system", {
+    name: req.user.name,
+    userGroup: req.user.group,
+    version: process.env[AppConstants.VERSION_KEY],
+    printerCount: printers.length,
+    page: "System",
+    octoFarmPageTitle: process.env[AppConstants.OCTOFARM_SITE_TITLE_KEY],
+    helpers: prettyHelpers,
+    clientSettings,
+    serverSettings,
+    systemInformation,
+    db: fetchMongoDBConnectionString(),
+    dashboardSettings: dashboardSettings,
+    serviceInformation: {
+      isDockerContainer: isDocker(),
+      isNodemon: isNodemon(),
+      isNode: isNode(),
+      isPm2: isPm2(),
+      update: softwareUpdateNotification
+    },
+    patreonData: require("../patreon.constants")
+  });
+});
 
```

Comment:
This was the only route change I made, I left all the endpoint cleaning up

Reply:
This is easy to convert. I thought the API specs were going to change for settings. Apparently not the case

Reply:
You said leave them so I did. The previous PR just cleaned up the routes. ie made them smaller URLs and more consistent.

Reply:
Nice, thanks for the helping hand

Reply:
> I did not test, and did not check ejs changes as its just too much. I trust you just chunked it up from the original file to children files.
> Nice work dude, another step towards V2 convertable code.
> 
> About filament: I'd appreciate if it there's as little backend changes as possible. I'd have to manually compare files and copy/paste/convert to ensure consistency. Very easy for me to make mistakes

It's only front end changes I'm meaning. I haven't linked the filament manager up to the new split off service file, not updated the sync buttons to use those changes.

Reply:
Hmm thought I'd replied to that elseware xD